CHANGELOG.md

```diff
@@ -0,0 +1,42 @@
+# Ignite UI for Angular Change Log
+
+All notable changes for each version of this project will be documented in this file.
+
+## 5.2.0
+- `igx-grid`
+    - Data-bound Input property `filtering` changed to `filterable`:
+
+    ```html
+    <igx-grid [data]="data">
+        <igx-column [field]="'ReleaseDate'" [header]="'ReleaseDate'"
+            [filterable]="true" dataType="date">
+        </igx-column>
+    </igx-grid>
+    ```
+
+    - @HostBinding `min-width` added to `IgxGridCellComponent` and `IgxGridHeaderCell`
```

Comment:
The `IgxGridCellComponent` no longer has a `value` setter, but instead has an `update(newValue)` modifier.